ocs_ci/helpers/dr_helpers_ui.py

```
@@ -955,9 +1153,48 @@ def navigate_using_fleet_virtualization(acm_obj):
+            )
+    if not nav_clicked:
+        # Fleet Virtualization perspective may have been renamed or its sidebar
+        # nav items reorganised; fall back to direct URL navigation.
+        log.info("VMs nav item not in sidebar, navigating to VMs page via URL")
+        current_url = acm_obj.driver.current_url
+        base_url = current_url.split("/k8s")[0].split("/multicloud")[0]
+        vms_url = (
+            f"{base_url}/k8s/all-clusters"
+            "/all-namespaces/kubevirt.io~v1~VirtualMachine"
+        )
+        log.info(f"Navigating to VMs URL: {vms_url}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)